compiler/damlc/tests/daml-test-files/QualifiedRetroactiveInterfaceInstance.EXPECTED.md

```diff
@@ -48,7 +48,7 @@
 >
 > * <a name="type-qualifiedretroactiveinterfaceinstance-transfer-84967"></a>**Choice** [Transfer](#type-qualifiedretroactiveinterfaceinstance-transfer-84967)
 >
->   Controller: newOwner, (DA.Internal.Record.getField @"owner" (view this))
+>   Controller: (DA.Internal.Record.getField @"owner" (view this)), newOwner
```

Comment:
We discussed removing the Internal getField last Monday at the Canton in-person meetup.

- A postprocessing step could detect these expression shapes `...getField @"<fieldName>" <expr>` and fix them back to `<expr>.<fieldName>`
- These blocks *should* have the original source locations embedded - if we could propagate them to this stage, then we could print out what the user originally typed.